shippable.yml

```diff
@@ -0,0 +1,34 @@
+language: go
+
+go:
```

Comment:
It's probably not necessary to change in this PR, but it's worth noting that in #5687 this is converting to using a build matrix to pass environment variables to control which API version is used. Is there a similar way to configure this in Shippable?